redhat-appstudio/service-provider-integration-operator
# Remove webhooks

### What does this PR do?
This is a followup PR of #80 that removes the rest of the webhooks from the codebase.

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/SVPI-75, https://issues.redhat.com/browse/SVPI-76


### How to test this PR?
The only functional change is that we no longer check that a user making updates to the `SPIAccessTokenBinding` is also able to get `SPIAccessToken`s. All other functionality should be unaffected.
